Autotenfication/Program.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+
+namespace Autotenfication
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            string login = "admin";
+            string password = "hello@123";
```

Comment:
@twinkleyarns You could change ";" to "," and remove all "string" from lines 10 11 12